scripts/build/sass-prefixer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * This is a helper function that prefixes css classes in javascript.
+ *
+ * @param {string} selector Selector being passed, e.g. '.m-datepicker .m-datepicker__check:after'
+ * @param {object} componentInfo Object containing componentName and prefixes
+ */
+export default function sassPrefixer(selector, componentInfo) {
```

Comment:
I personally dont like to have magic objects containing everything and nothing and is obfuscated. Better to clearly pass needed params as function paramters so that the function contract is defined clearly